app/views/dashboard/_moab_information.html.erb

```diff
@@ -2,20 +2,14 @@
 <p><strong class="fs-4">PreservedObject</strong> records store the druid and links to db records about both the Moab on local storage and all the replicated instances in the cloud.</p>
 <p><strong class="fs-4">MoabRecord</strong> records are about the Moab on storage.</p>
 <hr/>
-<div class="row row-cols-2">
-  <div class="column col-5">
-    <turbo-frame id="moab-record-versions" src="<%= dashboard_moab_record_versions_path %>" loading="lazy">
-      <%= render Spinner.new %>
-    </turbo-frame>
-  </div>
-  <div class="column col-7">
-    <turbo-frame id="moab-record-info" src="<%= dashboard_moab_record_info_path %>" loading="lazy">
-      <%= render Spinner.new %>
-    </turbo-frame>
-  </div>
-</div>
+<turbo-frame id="moab-record-versions" src="<%= dashboard_moab_record_versions_path %>" loading="lazy">
+  <%= render Spinner.new %>
+</turbo-frame>
+<hr/>
+<turbo-frame id="moab-record-info" src="<%= dashboard_moab_record_info_path %>" loading="lazy">
+  <%= render Spinner.new %>
+</turbo-frame>
```

Comment:
hrm.  I thought it was better use of screen real estate to have this tables beside each other.  Maybe it doesn't work with the left nav-tab stuff.

Reply:
can we have responsive tables and the side by side, or is that yucky?

Reply:
It turns out yucky because then you end up with a scroll bar on a small table.

Reply:
gotcha.